protocols.csv

```diff
@@ -24,6 +24,7 @@ code,	size,	name,	comment
 454,	0,	noise,
 460,	0,	quic,
 465,	0,	webtransport,
+466,    V,  certhash
```

Comment:
```suggestion
466,    V,	certhash,
```

maybe we need that verify.py script in here too

Reply:
https://github.com/multiformats/multicodec/blob/master/validate.py

Reply:
I was just using the GitHub web editor (the vscode in the browser), and it looked fine there. Maybe we need to adjust the formatting of this file, so this doesn't happen again in the future?